AlphaWallet/Tokens/ViewControllers/TokensCardViewController.swift

```diff
@@ -151,8 +150,10 @@ class TokensCardViewController: UIViewController, TokenVerifiableStatusViewContr
 
         if let selectedTokenHolder = selectedTokenHolder {
             let actions = viewModel.actions
-            buttonsBar.numberOfButtons = actions.count
+            buttonsBar.configuration = .combined(buttons: actions.count)
+            buttonsBar.optionButtons.first?.addTarget(self, action: #selector(optionsButtonTapped), for: .touchUpInside)
             buttonsBar.configure()
```

Comment:
It seems like, most of the time (except when there's a more button), these 2 lines either go together:

```
buttonsBar.configuration = .combined(buttons: actions.count)
buttonsBar.configure()
```

or there's a standalone call to `configure()`:

```
buttonsBar.configure()
```

- [ ] Do you think the 1st case is better like this. Less chance of setting `configuration` and forgetting to call `configure()`:


```
buttonsBar.configure(.combined(buttons: actions.count))
```

Make `configure()` accept an optional `ButtonsBarConfiguration` and assign it to `ButtonsBar.configuration` if non-nil?